runtime/compiler/trj9/z/codegen/J9CodeGenerator.cpp

```diff
@@ -104,6 +104,9 @@ J9::Z::CodeGenerator::CodeGenerator() :
    if (!comp->getOption(TR_FullSpeedDebug))
       cg->setSupportsDirectJNICalls();
 
+   if (cg->getSupportsVectorRegisters())
+      cg->setSupportsSIMDStringCaseConv();
```

Comment:
Did we check that `setSupportsVectorRegisters` is called before this point? It seems to be called in `TR::S390PrivateLinkage::S390PrivateLinkage`. We need to verify this is called before this point.